constants.js

```diff
@@ -10,3 +10,8 @@ export const incompatibleNames = {
   tiktok: 'tikTok',
   youtube: 'youTube',
 };
+
+export const flutterIncompatibleNames = {
+  ...incompatibleNames,
+  'color-filter': 'color-filter-ic'
```

Comment:
```suggestion
  'color-filter': 'color-filter-icon'
```

Sorry for the nitpick, but probably best to establish adding "Icon" to the end of incompatible names. And then this should be good to go!

Reply:
It's ok, I was unsure of what to call it 😅